source/1-SDLC-organization/lib/cdk-bootstrap-template.yml

```diff
@@ -745,6 +745,10 @@ Resources:
               - logs:CreateLogStream
               - logs:PutLogEvents
             Resource: !Sub "arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/codebuild/*"
+          - Effect: Allow
+            Action:
+              - cloudformation:DescribeStacks
```

Comment:
This is required for the CDK code to check the target environments. This check will be done by the CodeBuild step of the pipeline, so we need to extend the permission boundaries for the corresponding role.